From 403648bef0356b1aa7fe108eed37284483233368 Mon Sep 17 00:00:00 2001
From: Christopher Albert <albert@tugraz.at>
Date: Wed, 22 Apr 2026 12:28:04 +0200
Subject: [PATCH 16/23] testsuite: Add Bugzilla reporter attribution to
 gomp/pr107425.f90 [PR fortran/107425]

Retrospective attribution header added as part of a manual
Bugzilla-provenance audit on 2026-04-22 (meta-repo tracking commit
40f0a94; GitHub issues on krystophny/gcc-dev).  The existing GCC
Fortran testsuite convention (see pr113363.f90, pdt_50.f03,
arrayio_4.f90) uses an in-file "! Contributed by ..." line; practice
is inconsistent and attribution is not strictly required for small
Bugzilla reducers, but we are adding it retroactively for provenance
safety and completeness.  No behavioural change.

Assisted-by: Claude (Anthropic)
Assisted-by: GPT (OpenAI)

	PR fortran/107425

gcc/testsuite/ChangeLog:

	* gfortran.dg/gomp/pr107425.f90: Add reporter attribution header.

Signed-off-by: Christopher Albert <albert@tugraz.at>
---
 gcc/testsuite/gfortran.dg/gomp/pr107425.f90 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/testsuite/gfortran.dg/gomp/pr107425.f90 b/gcc/testsuite/gfortran.dg/gomp/pr107425.f90
index 77c31c20065..539d71c2d4d 100644
--- a/gcc/testsuite/gfortran.dg/gomp/pr107425.f90
+++ b/gcc/testsuite/gfortran.dg/gomp/pr107425.f90
@@ -2,6 +2,7 @@
 ! { dg-options "-fopenmp" }
 !
 ! PR fortran/107425
+! Contributed by G. Steinmetz  <gscfq@t-online.de>
 ! ICE in gimplify_var_or_parm_decl when an implicitly typed variable
 ! is used as a subscript in an iterator depend clause.
 
-- 
2.53.0

